rmgpy/data/kinetics/family.py

```diff
@@ -1219,13 +1219,30 @@ def applyRecipe(self, reactantStructures, forward=True, unique=True):
                     for i in range(4,highest+1):
                         atomLabels['*{0:d}'.format(i)].label = '*{0:d}'.format(4+highest-i)
                         
-            elif label == 'H_shift_cyclopentadiene':
+            elif label == 'Intra_ene_reaction':
                 # Labels for nodes are swapped
                 atomLabels['*1'].label = '*2'
                 atomLabels['*2'].label = '*1'
                 atomLabels['*3'].label = '*5'
                 atomLabels['*5'].label = '*3'
 
+            elif label == '6_membered_central_C-C_shift':
+                # Labels for nodes are swapped
+                atomLabels['*1'].label = '*3'
+                atomLabels['*3'].label = '*1'
+                atomLabels['*4'].label = '*6'
+                atomLabels['*6'].label = '*4'
+
+            elif label == '1,2_shiftC':
+                # Labels for nodes are swapped
+                atomLabels['*2'].label = '*3'
+                atomLabels['*3'].label = '*2'
+
+            elif label == 'Intra_R_Add_Exo_scission':
+                # Labels for nodes are swapped
+                atomLabels['*1'].label = '*4'
+                atomLabels['*4'].label = '*1'
+
```

Comment:
Is this section of applyRecipe even necessary? It just looks like atomLabels are defined for reactions that are their own reverse, but are then not used anywhere.